core/src/main/scala/com/salesforce/op/filters/RawFeatureFilter.scala

```diff
@@ -242,36 +243,64 @@ class RawFeatureFilter[T]
         Seq.fill(featureSize)(false)
       }
 
-    val distribMismatches =
+    case class DistributionMismatch(
```

Comment:
I'm not sure making a case class here is necessary if all it's used for is to pack/unpack those sequences in the calculation below. If there is a place you envision this being used elsewhere in the code, then it may make sense as its own case class (like FeatureDistribution or ExclusionReasons). If not, then it probably makes more sense to just do something like

`val (jsDivergences, fillRateDiffs, fillRatioDiffs) = ...`